docs/modules/examples/pages/advanced-rag.adoc

```diff
@@ -0,0 +1,798 @@
+parent_splitter = TokenTextSplitter(chunk_size=512, chunk_overlap=0)
+
+# Create a splitter for the child documents
+# Note: child documents should be smaller than parent documents
+child_splitter = TokenTextSplitter(chunk_size=128, chunk_overlap=0)
+
+# Create a parent document retriever
+parent_retriever = ParentDocumentRetriever(
+    vectorstore=vstore,
+    docstore=parent_store,
+    child_splitter=child_splitter,
+    parent_splitter=parent_splitter,
+)
+# Split and load the documents into the vector and parent stores
+parent_retriever.add_documents(docs)
+----
+
+== Base Retriever
+
+. As a control, first make a standard RAG pipeline.
++
+[source,python]
+----
+# Standard RAG, nothing fancy
+base_retriever = vstore.as_retriever()
+
+base_chain = (
+    {"context": base_retriever, "question": RunnablePassthrough()}
+    | prompt
+    | model
+    | StrOutputParser()
+)
+----
++
+. Run the queries on the `base_chain`:
++
+[tabs]
+======
+Python::
++
+[source,bash]
+----
+do_retrieval(base_chain)
+----
+
+Result::
++
+[source,console]
+----
+----------------------------------------
+Question: What motivates the narrator, Montresor, to seek revenge against Fortunato?
+
+Answer: The narrator, Montresor, seeks revenge against Fortunato because of
+the insults and injuries he has endured from him.
+
+Total Tokens: 852
+
+----------------------------------------
+Question: What are the major themes in this story?
+
+Answer: I don't know the answer.
+
+Total Tokens: 818
+
+----------------------------------------
+Question: What is the significance of the story taking place during the carnival season?
+
+Answer: The significance of the story taking place during the carnival season
+is not clear from the given context.
+
+Total Tokens: 832
+
+----------------------------------------
+Question: How is vivid and descriptive language used in the story?
+
+Answer: Vivid and descriptive language is used in the story to create a sense
+of atmosphere and to convey the intense emotions and actions of the
+characters.
+
+Total Tokens: 842
+
+----------------------------------------
+Question: Is there any foreshadowing in the story? If yes, how is it used in the story?
+
+Answer: Yes, there is foreshadowing in the story. The foreshadowing is used to
+hint at the fate of Fortunato and the revenge that the narrator plans
+to take. The mention of the "low moaning cry" from the depth of the
+recess and the "furious vibrations of the chain" suggest that
+something ominous and dangerous is happening to Fortunato. This
+foreshadows his eventual entrapment and demise in the catacombs.
+Additionally, the narrator's mention of wanting to "punish with
+impunity" and the idea of a wrong being unredressed when retribution
+overtakes its redresser foreshadow the narrator's plan to seek revenge
+on Fortunato without facing any consequences.
+
+Total Tokens: 974
+----
+======
++
+. We see some of the questions were answered well, and others were not.
```

Comment:
Change "We see some of the..." to "Some of the..." 

Probably my mistake of forgetting to change this text in the example to not use "We".